File: Programming with C#/1. C# Fundamentals I/5. Conditional Statements/01. Exchange If Greater/ExchangeIfGreater.cs
/*Problem 1. Exchange If Greater
Write an if-statement that takes two double variables a and b and exchanges their values if the first one is greater than the second one. As a result print the values a and b, separated by a space.*/

using System;

class ExchangeIfGreater
{
    static void Main()
    {
        Console.Title = "Exchange If Greater";

        Console.WriteLine("Enter two double variables: ");
        Console.WriteLine(new string('-', 40));
        Console.Write("First variable a is  --> ");
        double a = double.Parse(Console.ReadLine());
        Console.Write("Second variable b is --> ");
        double b = double.Parse(Console.ReadLine());
        Console.WriteLine(new string('-', 40));

        if (a > b)
        {
            Console.WriteLine("Result: {0} {1}", b, a);
        }
        else
        {
            Console.WriteLine("Result: {0} {1}", a, b);
        }

        Console.WriteLine(new string('-', 40));
    }
}

File: Web Design &UI/3. JS-Fundamentals/HW/03. Conditional statements/Conditional statements/exchangeIfGreater.js
/**
 Problem 1. Exchange if greater

 Write an if statement that takes two double variables a and b and exchanges their values if the first one is greater than the second.
 As a result print the values a and b, separated by a space.
 Examples:

 a    b    result
 5    2    2 5
 3    4    3 4
 5.5    4.5    4.5 5.5
 */

var arrayA = [5, 3, 5.5],
    arrayB = [2, 4, 4.5];

for (var i = 0; i < arrayA.length; i++) {
    isGreater(arrayA[i], arrayB[i]);
}

function isGreater(a, b) {
    if (a < b) {
        console.log(a + ' ' + b);
    } else {
        console.log(b + ' ' + a);
    }
}